### DISCUSSION TITLE
Delete a published package

### DISCUSSION THREAD

**Discussion by Skogsfrae** [onepub-dev/onepub#198] [Q&A] [answered]
Hi there,
I'm currently evaluating onepub and I was wondering whether it's possible to delete a published package.
I wanted to make a few tests before publishing a production package for my project, but couldn't find any documentation about it.

**Comment by bsutton** [comment]
The deletion of packages is something we are still debating internally so
at the moment you can't.

Can you provide some more details on your user case?

The normal path for experimentation is to give your package a dev label E.g.

0.1.0-dev.1

Would that work?

**Reply by Skogsfrae** [reply]
Well, since I didn't want to publish a production package for testing the service I created a dummy package (the tutorial one "calculator") just to see how it all works so I wasted one of the 15 private packages slot in the free plan.
Not a big deal for evaluation purposes, but in the past it happened to deprecate old libraries that weren't used anymore.
Since all the pricing plans have a storage limit it would be great to allow to delete some packages or even some old version of them.

**Comment by bsutton** [comment, marked as answer]
Reasonable points.

I will discuss then with the team.

**Reply by Skogsfrae** [reply]
Great!
Thanks for the quick response and the feedback

**Comment by nicks101** [comment]
I have a similar use case. Is there any update on "deleting a package" ?

**Comment by bsutton** [comment]
Yes you can now delete packages.

Under:  My OnePub | Manage Packages

Select the package and then click on the '...' icon (Additional Actions) and select Delete.

**Reply by nicks101** [reply]
Got it. Thanks for the quick reply.
